x-pack/plugins/upgrade_assistant/public/components/tabs/checkup/deprecations/reindex/flyout/progress.test.tsx

```diff
@@ -116,27 +132,51 @@ describe('ReindexProgress', () => {
     Array [
       Object {
         "status": "inProgress",
-        "title": "Pausing Machine Learning jobs",
+        "title": <FormattedMessage
+          defaultMessage="Pausing Machine Learning jobs"
+          id="xpack.upgradeAssistant.checkupTab.reindexing.flyout.checklistStep.reindexingChecklist.pauseMlStepTitle"
+          values={Object {}}
+        />,
       },
       Object {
         "status": "incomplete",
-        "title": "Setting old index to read-only",
+        "title": <FormattedMessage
+          defaultMessage="Setting old index to read-only"
+          id="xpack.upgradeAssistant.checkupTab.reindexing.flyout.checklistStep.reindexingChecklist.readonlyStepTitle"
```

Comment:
Similar to the question above: should we re-use or combine that fo `xpack.upgradeAssistant.checkupTab.reindexing.flyout.checklistStep.reindexingChecklist.readonlyStepTitle`

Reply:
See above.